### PR TITLE
[DeviceView] Fix `DeviceView` Error case for `command not found`

### PR DESCRIPTION
**Pull request by struss**
This commit will add handling code for 'command not found' in DeviceList script.

ONE-vscode-DCO-1.0-Signed-off-by: struss <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
this test will increase workflow TEM score.

**Review by llFreetimell — DISMISSED** [review]
LGTM

File: script/sdbSpecList.sh [hunk 1]
```diff
@@ -19,4 +19,14 @@
...
 #
 
-sdb devices | awk 'NR > 1 {if ($2 == "device") print $1}'
+SDB=$(which sdb)
+
+# Check SDB installed on this host.
+if [ -z "$SDB" ]
+then
+    # if not installed, return exitcode 127 which means `Command not found.`
+    echo "sdb command not found."
```
Comment (hyunsik-yoon) [inline script/sdbSpecList.sh:28]: I guess you need to echo to stderr (not stdout) to get the output in following handler:
```ts
      cmdSpawn.stderr.on('data', (data: any) => {
        error = result + data.toString();
        Logger.warn('DeviceList', error);
      });
```

**Review by hyunsik-yoon — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**